### PR TITLE
Mtb23/24 my story section

### PR DESCRIPTION
**Pull request by minna-le**
Hello, on this new PR, I 
1. Added an image grid on the right side of the My Story section.
2. Repositioned the left and right sides, setting the left section with flex: 1 and the right section with flex: 2 to align with your design layout.

### CODE REVIEW

File: styles/index.css [hunk 1]
```diff
@@ -2,28 +2,45 @@
...
+.left-corner h1 {
+  font-family: "Open Sans", sans-serif;
+  font-weight: 600;
 }
 
-.profile-pic:hover {
-  box-shadow: 0 0 2px 1px rgba(53, 55, 55, 0.5);
+.myStory-wrapper {
+  background-color: rgb(253, 238, 240);
+  pad: 40px;
```
Comment (Truniole) [inline styles/index.css]: typo?

File: styles/index.css [hunk 2]
```diff
@@ -2,28 +2,45 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
-
-  padding: 60px 20px;
-  gap: 90px;
+  background-color: white;
+  padding: 20px 20px;
+  gap: 70px;
   max-width: 800px;
```
Comment (Truniole) [inline styles/index.css]: Let's use 1500px max-width

File: styles/index.css [hunk 3]
```diff
@@ -2,28 +2,45 @@
...
+  justify-content: space-between;
+  gap: 10px;
 }
 
-.blogs-section h1,
-.blogs-section p {
-  justify-content: center;
+.under-image {
+  /* 1/3 33% - gap(10px) */
+  width: calc(33.333% - 10px);
```
Comment (Truniole) [inline styles/index.css:41]: This is great 👏

File: styles/index.css [hunk 4]
```diff
@@ -2,28 +2,45 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
-
-  padding: 60px 20px;
-  gap: 90px;
+  background-color: white;
+  padding: 20px 20px;
```
Comment (Truniole) [inline styles/index.css]: Let's add a bit more horizontal padding ~48px?

**Review by Truniole — CHANGES_REQUESTED** [review]

File: styles/index.css [hunk 5]
```diff
@@ -2,28 +2,45 @@
...
   justify-content: space-between;
   align-items: center;
-
-  padding: 60px 20px;
-  gap: 90px;
-  max-width: 800px;
+  background-color: white;
+  padding: 20px 48px;
+  gap: 70px;
+  max-width: 1500;
```
Comment (Truniole) [inline styles/index.css]: 1500px the code editor won't complain on this but the browser can't interpret raw numerical value

File: styles/index.css [hunk 6]
```diff
@@ -2,28 +2,45 @@
...
+}
+.left-corner h1 {
+  font-family: "Open Sans", sans-serif;
+  font-weight: 600;
 }
 
-.profile-pic:hover {
-  box-shadow: 0 0 2px 1px rgba(53, 55, 55, 0.5);
+.myStory-wrapper {
+  background-color: rgb(253, 238, 240);
```
Comment (Truniole) [inline styles/index.css]: Can we move this color to maybe white instead

File: styles/index.css [hunk 7]
```diff
@@ -2,28 +2,45 @@
...
+  padding: 40px;
+}
+.secondrow-images {
+  display: flex;
+  justify-items: center;
```
Comment (Truniole) [inline styles/index.css]: no need to `justify-items` here

File: styles/index.css [hunk 8]
```diff
@@ -2,28 +2,45 @@
...
+  justify-content: space-between;
+  gap: 10px;
 }
 
-.blogs-section h1,
-.blogs-section p {
-  justify-content: center;
+.under-image {
+  /* 1/3 33% - gap(10px) */
```
Comment (Truniole) [inline styles/index.css]: remove

**Review by Truniole — CHANGES_REQUESTED** [review]

File: styles/index.css (hunk 5 above)
Comment (minna-le) [reply styles/index.css]: yes, I will add px

File: styles/index.css (hunk 6 above)
Comment (minna-le) [reply styles/index.css]: Yes, lets do that

**Review by Truniole — APPROVED** [review]

**Pull request merged**